pages/producten/index.vue

```diff
@@ -0,0 +1,35 @@
+<style src="./producten.css" scoped/>
+<template>
```

Comment:
It would be nice to have a header with the name of this page here
![image](https://github.com/GyKevin/voedselbank/assets/30209145/e9781b2a-d09e-4751-8569-5a2fd7e85029)